### DISCUSSION TITLE
colormap_name for MosaicJSON

### DISCUSSION THREAD

**Discussion by giswqs** [developmentseed/titiler#440] [Q&A] [answered]
Is it possible to set the `colormap_name` param for MosaicJSON? The following script can render a MosaicJSON as a gray-scale image. However, when the `colormap_map` is set, the tile layer does not show up. 

```python
import requests
from folium import Map, TileLayer

titiler_endpoint = "https://titiler.xyz"  

r = requests.get(
    f"{titiler_endpoint}/mosaicjson/tilejson.json",
    params={
        "url": "https://fldpln.blob.core.windows.net/maps/spring.json.gz",
        # 'colormap_name': "blues"
    }
).json()
print(r)

m = Map(
    location=(36.8608, -94.8152),
    zoom_start=13
)

tiles = TileLayer(
    tiles=r["tiles"][0],
    min_zoom=r["minzoom"],
    max_zoom=r["maxzoom"],
    opacity=1,
    attr="NOAA Esri"
)

tiles.add_to(m)
m
```
![image](https://user-images.githubusercontent.com/5016453/155610147-bb62ee51-f6d3-4b67-a897-f6de09845dfb.png)

**Comment by vincentsarago** [comment, marked as answer]
sorry @giswqs I moved this to `discussion` because there is no `bug` here.

I think you're issue is you need to add a `rescale` parameter to rescale the data from 0 to 255. 

![](https://user-images.githubusercontent.com/10407788/155611386-3d7c983d-178e-405d-bba9-11be647aaff4.png)

**Reply by giswqs** [reply]
Awesome! Many thanks for the prompt response.
